Debian/Dockerfile

```diff
@@ -202,12 +198,21 @@ RUN make -j 8
 # need to run it once as root to link up libs
 RUN src/imaptest || true
 
-WORKDIR /srv/cyruslibs.git
-RUN git fetch
-RUN git checkout origin/master
-RUN git submodule init
-RUN git submodule update
-RUN ./build.sh
+WORKDIR /srv
+RUN <<EOF
+set -e
+git config --global http.sslverify false
+git clone --depth 1 --shallow-submodules -b cyruslibs-fastmail-v58 \
+  https://github.com/cyrusimap/cyruslibs.git cyruslibs.git
+cd cyruslibs.git
+git fetch
```

Comment:
This sets .git/FETCH_HEAD, is that necessary? It's very minor but is a network callout...

Reply:
Doing a build without this.  It seems unneeded.